sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/schema/cqlfirst/ddl/fetchers/AllKeyspacesFetcher.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.graphql.schema.cqlfirst.ddl.fetchers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.sgv2.graphql.schema.CassandraFetcher;
+import io.stargate.sgv2.graphql.web.resources.StargateGraphqlContext;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AllKeyspacesFetcher extends CassandraFetcher<List<KeyspaceDto>> {
+
+  @Override
+  protected List<KeyspaceDto> get(
+      DataFetchingEnvironment environment, StargateGraphqlContext context) {
+    return context.getBridge().getAllKeyspaces().stream()
```

Comment:
I did, but I had to [abandon the idea](https://github.com/stargate/stargate/pull/544#issuecomment-758786043) because of how the v1 persistence is implemented.

The v2 architecture will make it possible again, as long as we have async methods on `StargateBridgeClient` (as a matter of fact, I'm working on that in another PR). I will reopen #421.